Revert "Pango_font: Free temporary font description"
This reverts commit 102bed8b5a6332f2021a284bd5a937a2603fb327.
diff --git a/lily/pango-font.cc b/lily/pango-font.cc
index 053ce17143..1a63b18338 100644
--- a/lily/pango-font.cc
+++ b/lily/pango-font.cc
@@ -385,7 +385,6 @@ Pango_font::pango_item_string_stencil (PangoGlyphItem const *glyph_item) const
   PangoFontDescription *descr = pango_font_describe (pa->font);
   Real size = pango_font_description_get_size (descr)
               / (static_cast<Real> (PANGO_SCALE));
-  pango_font_description_free (descr);
 
   if (ps_name_str0.empty ())
     warning (_f ("no PostScript font name for font `%s'", file_name));
